packages/lix-sdk/src/database/index.ts

```diff
@@ -1,5 +1,7 @@
 export * from "./schema.js";
 export * from "../key-value/database-schema.js";
-export { jsonObjectFrom, jsonArrayFrom } from "kysely/helpers/sqlite";
+// Replace the kysely helpers with our own implementations for JSONB
+export { jsonObjectFrom, jsonArrayFrom, jsonb } from "./json.js";
```

Comment:
The `jsonObjectFrom` and `jsonArrayFrom` are likely not needed. The SQLite functions used in the `json_array()` Kysely helper should work on jsonb as well. 

If not, all up for having our own jsonObjectFrom and jsonArrayFrom functions.

Reply:
true 👍